explore/src/main/scala/explore/targeteditor/SpectralDefinitionEditor.scala

```diff
@@ -197,7 +198,13 @@ sealed abstract class SpectralDefinitionEditorBuilder[
         props.catalogInfo.flatMap(ci =>
           ci.objectType.map(ot =>
             React.Fragment(
-              FormLabel(htmlFor = "catalogInfo".refined)(ci.catalog.shortName),
+              FormLabel(htmlFor = "catalogInfo".refined)(
```

Comment:
Just a note. You can pass the label with icon directly to FormInputText by passing it as a Fragment instead of wrapping the label and input in a Fragment.

Reply:
I considered that, but does that render the same structure? This is in the context of a `grid` and we must keep the html structure. For some reason, we are using `FormLabel` for the other fields here, so I went with that.

In fact, I broke the brightness table, which is now pushed to the bottom. I will fix in a future PR.

Reply:
I think it keeps the same structure. But, it's not a big deal to do it either way.

Reply:
Ah ok, the case where we have an explicit `FormLabel` is an `EnumDropdown`, which doesn't have a `label`. I'll change it in the next PR.